.github/workflows/imagebuild.yml

```diff
@@ -0,0 +1,131 @@
+name: Build and Push Georust Docker Images
+
+on:
+  push:
+    branches:
+      - main
+      - staging
+      - trying
+  pull_request:
+  merge_group:
+
+env:
+  LIBPROJ_VERSION: 9.4.0
+  MAIN_IMAGE_NAME: libproj-builder
+
+jobs:
+  build_main_image:
+    name: Build the main image for geo and proj
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        rust_version: [1.77]
```

Comment:
I wonder if we actually need to specify rust_version for the libprojbuilder - it's not even using rust, right? It's fine to leave it this way for now, since that's how it's always been, but worth considering for future simplification. It'd speed up the build a bit and avoid having to track supported versions in two places.

Reply:
Oooh. That's a good point.

Reply:
The `libproj-builder` Docker image does expect a Rust version:

`ARG RUST_VERSION`

Unless I'm misunderstanding something about how the multi-stage builds work?